### ISSUE TITLE
disconnecting client as received unexpected packet type while expected request handshake

### ISSUE THREAD

**Issue by Azelf999** [TLeonardUK/ds3os#40]
Hello, I have configured the server with public ip in ServerHostname, private in ServerPrivateHostname and opened the router ports (50000-50010-50020-50050) with both TCP and UDP. My friends (outside my network) and I open the files "as administrator", but only I can connect. When they try to connect in the prompt it shows the error "disconnecting client as received unexpected packet type while expected request handshake".
![Cattura](https://user-images.githubusercontent.com/98402726/150989701-bfff66e1-c19c-4849-98a2-c0b44692b879.PNG)

**Comment by TLeonardUK** [comment]
Looks like its sending something prior to the expected handshake flow, thats a new one, I've never seen that occur before. 

Does this happen everytime they try to connect?

Can they connect to the ?ServerName? - thats the one I'm running.

**Comment by Azelf999** [comment]
Yes, it happens everytime. We tried to connect to other open servers and succeeded without any problems (including ?ServerName?).

**Comment by Azelf999** [comment]
I've update to v.13.0.0. Now show this Warning:

![image](https://user-images.githubusercontent.com/98402726/152406845-959cd244-9ea7-438f-9620-2d83cffada05.png)

**Comment by TLeonardUK** [comment]
Weird question, you haven't changed any of the auth/game/login ports in the config file have you?

Thats super bizzare, that package type is something that only makes sense to send to the login server, sending it to the auth server makes no sense 🤔

**Comment by Azelf999** [comment]
Nope, I only changed the necessary settings (the two IPs and the server name).
